internal/expr/prepare.go

```diff
@@ -291,3 +418,32 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 
 	return &PreparedExpr{inputs: inputs, outputs: outputs, sql: sql.String()}, nil
 }
+
+// printCols prints a bracketed, comma seperated list of fullNames.
+func printCols(cs []fullName) string {
```

Comment:
To me, `print` in the name of this function implies that `fmt.Print` or similar is called. Perhaps this could be renamed to something like `formatColNames`?

Reply:
Good point, changed.